README.md

```diff
@@ -59,8 +59,78 @@ rdataretriever::download('portal', './data/')
 portal = rdataretriever::fetch('portal')
 names(portal)
 head(portal$species)
+
 ```
 
+New Spatial data Installation
+-----------------------------
+
+**Set-up and Requirements(Tools)**
+
+**Tools**
+
+-  psql, raster2pgsql, shp2pgsql, dgal,
+
+The `rdataretriever` supports installation of spatial data into `Postgres DBMS`.
+
+1. **Install Postgres**
+
+	To install `Postgres` from scratch please ref to [Postgres installation](https://trac.osgeo.org/postgis/wiki/UsersWikiPostGIS21UbuntuPGSQL93Apt).
```

Comment:
"To install `PostgreSQL` for use with spatial data please refer to the [OSGeo Postgres installation instructions]..."